### DISCUSSION TITLE
Does this provide unlimited flexibility?

### DISCUSSION THREAD

**Discussion by jamesone** [Gusto/buildkite-builder#96] [General]
I am wondering whether the buildkite-builder allows you to build any pipeline you want? Are there any limits to what you can do?

**Comment by technicalpickles** [comment]
buildkite-builder is ultimately generating a pipeline.yaml and uploading that to Buildkite. So, any pipeline you could build with YAML you should be able to do with buildkite-builder.

**Reply by jamesone** [reply]
thanks for your reply!
